tests/test_validate.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf8 -*-
```

Comment:
With you coverage only gets better! 📈 

I realize coveralls can be annoying when it doesn't pass the build just because the coverage decreased 0.01% 😆  - but if this gets on your way again you could also change [the threshold settings](https://coveralls.io/github/artefactual-labs/mets-reader-writer/settings). I'm not sure what the defaults are but they're small!

![image](https://user-images.githubusercontent.com/606459/32691208-a486dd48-c6b8-11e7-8de2-d42acd56bd0e.png)

Reply:
Good to know about the threshold settings. The test coverage actually decreased by about 0.5%, but yes it was annoying to have to add more tests to get the CI checks to pass.